inspire/testsuite/test_ICU.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2015 CERN.
+#
+# INSPIRE is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+from invenio.testsuite import InvenioTestCase
+
+
+class IcuPluginTests(InvenioTestCase):
+
+    def test_icu(self):
+        from invenio_ext.es import es
+        self.assertTrue('analysis-icu' in es.cat.plugins())
```

Comment:
To be more informative, maybe add a `msg='The ICU analyzer plugin is not installed. Please add it by following the instructions at https://github.com/elastic/elasticsearch-analysis-icu'` argument?